### PR TITLE
Change Python package from seamapi to seam. Get rid of yarn install.

### PR DESCRIPTION
**Pull request by DebbieAtSeam**

### CODE REVIEW

File: docs/seam-components/make-a-supported-devices-page-with-react.md [hunk 1]
```diff
@@ -14,8 +14,6 @@ This guide will show you how to drop [`<SupportedDeviceTable />`](https://react.
 
 ```bash
 npm install --save @seamapi/react
-# or via yarn
-yarn add @seamapi/react
```
Comment (sybohy) [inline docs/seam-components/make-a-supported-devices-page-with-react.md:18]: any reason why we're dropping yarn support?

File: docs/thermostats/get-thermostat.md [hunk 2]
```diff
@@ -25,7 +25,8 @@ Retrieves the details of a thermostat.
 
 {% tabs %}
 {% tab title="Python" %}
-<pre class="language-python"><code class="lang-python">from seamapi import Seam
+```javascript
```
Comment (sybohy) [inline docs/thermostats/get-thermostat.md:28]:
```suggestion
```python
```

**Review by sybohy — CHANGES_REQUESTED** [review]
Made a couple of changes; need to wait for @seveibar to deploy the python lib to the new name before we can merge this in

File: docs/seam-components/make-a-supported-devices-page-with-react.md (hunk 1 above)
Comment (DebbieAtSeam) [reply docs/seam-components/make-a-supported-devices-page-with-react.md:18]: @sybohy https://hello-seam.slack.com/archives/C01D0J9ACLF/p1705301765773369

Comment (DebbieAtSeam) [reply docs/seam-components/make-a-supported-devices-page-with-react.md:18]: and https://hello-seam.slack.com/archives/C01D0J9ACLF/p1706726806980289

**Pull request closed without merging**